crates/cheatnet/src/state.rs

```diff
@@ -140,8 +140,9 @@ pub enum CheatStatus<T> {
 }
 
 /// Tree structure representing trace of a call.
+#[derive(Serialize, Deserialize)]
 pub struct CallTrace {
-    pub entry_point: CallEntryPoint,
+    pub entry_point: crate::blockifier_structs::CallEntryPoint,
```

Comment:
Does this have to be in other type? I think it could be converted only on dump request